packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -732,6 +788,13 @@ class SwipeableViews extends Component {
       springConfig,
       style,
       threshold, // eslint-disable-line no-unused-vars
+      onTouchStart, // eslint-disable-line no-unused-vars
+      onTouchEnd, // eslint-disable-line no-unused-vars
+      onMouseDown, // eslint-disable-line no-unused-vars
+      onMouseLeave, // eslint-disable-line no-unused-vars
+      onMouseMove, // eslint-disable-line no-unused-vars
+      onMouseUp, // eslint-disable-line no-unused-vars
+      enableMouseEvents, // eslint-disable-line no-unused-vars
```

Comment:
I guess these 'onTouch/Mouse' handlers don't need to be remove here because they will simply get overwritten right?